packaging/flavorgen/flavors/kubevip/topology.go

```diff
@@ -96,11 +96,11 @@ func TopologyPatch() clusterv1.ClusterClassPatch {
 			Path:      "/spec/template/spec/kubeadmConfigSpec/preKubeadmCommands/-",
 			ValueFrom: &clusterv1.JSONPatchValue{Template: ptr.To("/etc/kube-vip-prepare.sh")},
 		},
-		clusterv1.JSONPatch{
-			Op:        "add",
-			Path:      "/spec/template/spec/kubeadmConfigSpec/postKubeadmCommands/-",
-			ValueFrom: &clusterv1.JSONPatchValue{Template: ptr.To("/etc/kube-vip-cleanup.sh")},
-		},
+		// clusterv1.JSONPatch{
+		// 	Op:        "add",
+		// 	Path:      "/spec/template/spec/kubeadmConfigSpec/postKubeadmCommands/-",
+		// 	ValueFrom: &clusterv1.JSONPatchValue{Template: ptr.To("/etc/kube-vip-cleanup.sh")},
+		// },
 	)
```

Comment:
Hey @chrischdi ,

Are the commented code left intentionally? Otherwise, I think we could remove them :-)

Reply:
Good catch. I agree => deleted